src/views/MarketsDropdown.tsx

```
@@ -47,335 +47,338 @@ import { MarketFilter } from './MarketFilter';
+                                        <div tw="flex flex-col text-start">
+                                            <span tw="font-mini-book">
+                                                {stringGetter({ key: STRING_KEYS.NOT_LAUNCHED })}
+                                            </span>
+                                            <h2 tw="mt-[-0.25rem] text-color-text-2 font-medium-medium">
+                                                {currentMarketId}
+                                            </h2>
+                                        </div>
+                                    </>
+                                ) : (
+                                    <div tw="flex items-center gap-0.25">
+                                        <$AssetIconWithStar>
```

ESLint

[error] Replace `········································` with `····················`

(prettier/prettier)